ops/model.py

```diff
@@ -1045,6 +1049,17 @@ def pebble(self) -> 'pebble.Client':
         """The low-level :class:`ops.pebble.Client` instance for this container."""
         return self._pebble
 
+    @property
+    def ready(self) -> bool:
```

Comment:
Assuming we add this, I'd prefer this to be spelled `is_ready()` as a normal function. My rule of thumb is that properties should be things that are fairly cheap to calculate locally, and don't raise exceptions. Whereas this does a network request and may raise other exceptions like `APIError`.